.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Provide a code example and any sample input data (e.g. an H5AD) as an attachment to reproduce this behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Desktop (please complete the following information):**
+ - TileDB-SOMA version:
```

Comment:
Let's show people _how_ to get this:

```
import tiledbsoma, tiledb, sys
tiledbsoma.__version__
tiledb.__version__
sys.version_info
```

Reply:
Actually I made an adjustment to make this language agnostic. Your comment made me realize that this bug report is only designed for Python and we want it to be applicable for any language. So I left off any instructions and generalized it.

Reply:
OK, but let's keep our language inclusive for folks who are primarily computational biologists & not primarily software people. Bug reports are the "interface" or "membrane" between app users & engineers, and this document is written by the latter but it needs to be friendly and non-intimidating for the former.

Reply:
TBH @maniarathi I'd be happier if we included some helpful instructions for R and Python both.

Reply:
```
utils::packageVersion("tiledbsoma")
utils::packageVersion("tiledb")
R.Version()$version.string
```

Reply:
Sounds good; could you suggest an edit please on the language? And I am not familiar with R so could you please suggest the instruction set for R?

Reply:
@maniarathi I just did ☝️

Reply:
**Versions (please complete the following information):**
 - TileDB-SOMA version:
 - Language and language version (e.g. Python 3.8, R 4.1):
 - OS (e.g. MacOS, Ubuntu Linux): 

Reference info for Python:

```
import tiledbsoma, tiledb, sys
tiledbsoma.__version__
tiledb.__version__
sys.version_info
```

Reference info for R:

```
library(tiledbsoma)
library(tiledb)
utils::packageVersion("tiledbsoma")
utils::packageVersion("tiledb")
R.Version()$version.string
```

Reply:
Also:   `tiledb::tiledb_version(TRUE) ` which can vary for `tiledb` installation (the pinning is soft).

But my vote would be to keep the template _shorter_, omit this and add a helper function (in both language variants) named, say, `tiledb_soma_versions()` or alike.

Reply:
Also, confusingly, there are umpteen ways to ask R itself for its version and `R.version.string` may be simpler than John's `R.Version()$version.string`.

Reply:
I agree that a shorter template is better because otherwise the instructions on how to get the version shows up in every Github issue. I wonder if a better solution might be to use Github Discussions on this repo and have a Q&A one of which can be "How do I get the version numbers".

Reply:
How about my earlier suggestion to have a helper function?  Also, when one use `tiledb` and loads it, we greet -- this is done by a few other R packages too and helpful in the context of external libraries and their versions:

```r
> library(tiledb)
TileDB R 0.18.0.1 with TileDB Embedded 2.15.0. See https://tiledb.com for more information.
> 
```

(I tend to build of our 'dev' hence 2.15.0 for me.)

Reply:
I've already accepted.

I'm advocating for inclusivity. I'm also sensitive to brevity concerns.

A helper function I think would be a good happy medium.

Reply:
@eddelbuettel Sounds good on the helper function! Would you mind creating an issue so that we can prioritize creating it? And then I will cut the content from this template.

Reply:
Sure, done as #851

Reply:
Thank you! @eddelbuettel